PageObjectPattern/src/test/java/utils/TestDataGenerator.java

```diff
@@ -0,0 +1,30 @@
+package utils;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Random;
+
+public class TestDataGenerator {
+
+	// Method to create random email address
+	public String getRandomEmail() {
+		String staticName = "ytestautomation@mailinator.com";
+		Random rand = new Random();
+		int rand_number = rand.nextInt(1000);
+		String randomEmail = rand_number + staticName;
+		return randomEmail;
+	}
+
+	// Method to save Email to file
+	public void saveEmail(String UserEmail) throws IOException {
+		String file = "C:\\Users\\Yulia_Borisova\\Desktop\\Java_Study\\Projects\\Eclipse-Selenium\\PageObjectPattern\\src\\test\\resources\\testdata\\CreatedUserEmails";
```

Comment:
File location can be simplified to a relative path rather than absolute path.

Reply:
done